examples/bvh_driver/bvh_driver.cpp

```diff
@@ -245,6 +245,38 @@ class KokkosScopeGuard
       ->UseManualTime()                                                        \
       ->Unit(benchmark::kMicrosecond);
 
+// NOTE Motivation for this class that stores the argument count and values is
+// I could not figure out how to make the parser consume arguments with
+// Boost.Program_options
+class CmdLineArgs
+{
+  int argc_;
+  std::vector<char *> argv_;
```

Comment:
Error prone way to set something private. Also, I thought that private variables started with an `_`

Reply:
Why are using raw pointers?

Reply:
Because you can't get a `char **` with a vector of strings